plugins/targetmanagers/targetlist/targetlist.go

```diff
@@ -84,13 +86,14 @@ func (t TargetList) ValidateReleaseParameters(params []byte) (interface{}, error
 }
 
 // Acquire implements contest.TargetManager.Acquire
-func (t *TargetList) Acquire(ctx context.Context, jobID types.JobID, parameters interface{}, tl target.Locker) ([]*target.Target, error) {
+func (t *TargetList) Acquire(ctx context.Context, jobID types.JobID, jobTargetManagerAcquireTimeout time.Duration, parameters interface{}, tl target.Locker) ([]*target.Target, error) {
 	acquireParameters, ok := parameters.(AcquireParameters)
 	if !ok {
 		return nil, fmt.Errorf("Acquire expects %T object, got %T", acquireParameters, parameters)
 	}
 
-	if err := tl.Lock(jobID, acquireParameters.Targets); err != nil {
+	lockTimeout := jobTargetManagerAcquireTimeout + config.LockRefreshTimeout
```

Comment:
You moved this inside the locker, right? (Adding the refresh timeout)

Reply:
Reverted this, it was probably an experiment leftover.